### ISSUE TITLE
log warning messages on DataFrame

### ISSUE THREAD

**Issue by zqian** [tl-its-umich-edu/my-learning-analytics#1070]
With MyLA-Fall-2020.03.01 release, there are many DataFrame warning messages in the log file:

/usr/local/lib/python3.8/site-packages/pandas/core/frame.py:2963: SettingWithCopyWarning:
```
 A value is trying to be set on a copy of a slice from a DataFrame.
 Try using .loc[row_indexer,col_indexer] = value instead
  
 See the caveats in the documentation: https://pandas.pydata.org/pandas-docs/stable/user_guide/indexing.html#returning-a-view-versus-a-copy
 self[k1] = value[k2]
 /code/dashboard/views.py:579: SettingWithCopyWarning:
 A value is trying to be set on a copy of a slice from a DataFrame.
 Try using .loc[row_indexer,col_indexer] = value instead
  
 See the caveats in the documentation: https://pandas.pydata.org/pandas-docs/stable/user_guide/indexing.html#returning-a-view-versus-a-copy
 df3['graded'] = df3['graded'].fillna(False)
 /code/dashboard/views.py:581: SettingWithCopyWarning:
 A value is trying to be set on a copy of a slice from a DataFrame.
 Try using .loc[row_indexer,col_indexer] = value instead
  
 See the caveats in the documentation: https://pandas.pydata.org/pandas-docs/stable/user_guide/indexing.html#returning-a-view-versus-a-copy
 df3['percent_gotten'] = df3.apply(lambda x: user_percent(x), axis=1)
 /code/dashboard/views.py:582: SettingWithCopyWarning:
 A value is trying to be set on a copy of a slice from a DataFrame
  
 See the caveats in the documentation: https://pandas.pydata.org/pandas-docs/stable/user_guide/indexing.html#returning-a-view-versus-a-copy
 df3.sort_values(by=['graded', 'due_date_mod'], ascending=[False, True], inplace=True)
 /usr/local/lib/python3.8/site-packages/pandas/core/frame.py:3990: SettingWithCopyWarning:
 A value is trying to be set on a copy of a slice from a DataFrame
  
 See the caveats in the documentation: https://pandas.pydata.org/pandas-docs/stable/user_guide/indexing.html#returning-a-view-versus-a-copy
 return super().drop(
 /usr/local/lib/python3.8/site-packages/pandas/core/frame.py:2963: SettingWithCopyWarning:
 A value is trying to be set on a copy of a slice from a DataFrame.
 Try using .loc[row_indexer,col_indexer] = value instead
  
 See the caveats in the documentation: https://pandas.pydata.org/pandas-docs/stable/user_guide/indexing.html#returning-a-view-versus-a-copy
 self[k1] = value[k2]
 /code/dashboard/views.py:579: SettingWithCopyWarning:
 A value is trying to be set on a copy of a slice from a DataFrame.
 Try using .loc[row_indexer,col_indexer] = value instead
  
 See the caveats in the documentation: https://pandas.pydata.org/pandas-docs/stable/user_guide/indexing.html#returning-a-view-versus-a-copy
 df3['graded'] = df3['graded'].fillna(False)
 /code/dashboard/views.py:581: SettingWithCopyWarning:
 A value is trying to be set on a copy of a slice from a DataFrame.
 Try using .loc[row_indexer,col_indexer] = value instead
  
 See the caveats in the documentation: https://pandas.pydata.org/pandas-docs/stable/user_guide/indexing.html#returning-a-view-versus-a-copy
 df3['percent_gotten'] = df3.apply(lambda x: user_percent(x), axis=1)
 /code/dashboard/views.py:582: SettingWithCopyWarning:
 A value is trying to be set on a copy of a slice from a DataFrame
  
 See the caveats in the documentation: https://pandas.pydata.org/pandas-docs/stable/user_guide/indexing.html#returning-a-view-versus-a-copy
 df3.sort_values(by=['graded', 'due_date_mod'], ascending=[False, True], inplace=True)
 /usr/local/lib/python3.8/site-packages/pandas/core/frame.py:3990: SettingWithCopyWarning:
 A value is trying to be set on a copy of a slice from a DataFrame
  
 See the caveats in the documentation: https://pandas.pydata.org/pandas-docs/stable/user_guide/indexing.html#returning-a-view-versus-a-copy
 return super().drop(
```

**Comment by jonespm** [comment]
`a3afd2ae (Pushyami Gundala            2018-11-20 10:48:09 -0500 582)     df3.sort_values(by=['graded', 'due_date_mod'], ascending=[False, True], inplace=True)
`



Matthew Jones  44

**Comment by jonespm** [comment]
This was fixed by #1078
